### PR TITLE
Delete temp file created locally after uploading to cloud service

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

File: services/util/src/services/downloads.js [hunk 1]
```diff
@@ -52,6 +54,14 @@ module.exports = (DownloadsPersistence, CloudServices) => {
         uploaded.Location,
         uploaded.exp_date,
       );
+
+      fs.unlink(file.path, (err) => {
+        if (err) {
+          const Error = { message: 'Error by deleting temp file' };
+          throw Error;
```
Comment (cazapatamar) [inline services/util/src/services/downloads.js:61]: @erikasv please help me out to review how to handle error in this callback by deleting temp file. Let me know your opinion of this implementation.

**Review by cazapatamar — COMMENTED** [review]
@erikasv please help me out to review how to handle error in this callback by deleting temp file. Let me know your opinion of this implementation.

File: services/util/src/services/downloads.js (hunk 1 above)
Comment (erikasv) [reply services/util/src/services/downloads.js:61]: hmm, I don't think you should throw it, better just log the error. It's ok that is an async call and better not affect the rest of the flow, if it doesn't get deleted it's not a major problem (unless it happens a lot xD)

Comment (erikasv) [reply services/util/src/services/downloads.js:61]: Also I would remove the `by` in the message

**Review by erikasv — CHANGES_REQUESTED** [review]
Please change "before" for "after" in the PR description

**Review by erikasv — APPROVED** [review]

**Pull request merged**